### DISCUSSION TITLE
What's wrong?

### DISCUSSION THREAD

**Discussion by Firstofficerjose** [flybywiresim/aircraft#3429] [Help]
So often I fly the A320 Neo and have a FPS problem during the flight development (Cruise phase, decent and landing). I am on the most recent update of the Airbus 320 Neo, and I don’t know what happens that I lose too much fps and I am using a poor configuration, with even ground level detail at 30 and all off in the MSFS configuration. I don’t understand what happens since my PC is decent I will leave the specifications here, someone knows? because I got so low fps during those phases I have received up to 10fps and even 7-9 during landing which makes the gaming experience horrible, before this never had happened to me, my sim used to ran up to a maximum of 50-60fps even using the Airbus 320 Neo in big cities like New York, Miami, Los Angeles. How can I solve this problem if I already have everything at low quality, I cannot continue or get a better quality since I have everything at low performance. Can you help me?

PC SPECS-
GE FORCE RTX 2070 SUPER
DRIVER VERSION 461.40
AMD RYZEN 7 3700X 8-Core Processor
15.93 GB RAM
1929 x 1080, 60HZ

**Comment by robinottawa** [comment]
Do you mean the NX and not the Neo? 

I've had some problems with frames where an update reset everything, but it sounds like you double checked everything. I also had trouble because of a broken wifi connection. Have you checked that?

**Comment by MapleNAV-TCASFAULT** [comment]
Future performance updates will fix this issue as this issue is most likely the same as #3399 . The only way to get better fps atm is to use the stable version.
